### PR TITLE
fix customization xml

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

fix customization to move new schedule namespace under `Revit`.

### Reviewers

@QilongTang 

### FYIs

@ksobon @jnealb

### CODE REVIEW

**Comment by ksobon** [comment]
did I screw this one up?

**Comment by mjkkirschner** [comment]
nope, I forgot to mention this, this change sticks your new namespace under the revit cateogory instead of exposing the RevitNodes.Revit.Schedules category.

**Comment by QilongTang** [comment]
Cool. Did not know that we have our own customized mapping of namespaces. LGTM.

File: src/Libraries/RevitNodes/RevitNodes_DynamoCustomization.xml [hunk 1]
```diff
@@ -19,6 +19,9 @@
     <namespace name="Revit.Filter">
       <category>Revit.Filter</category>
     </namespace>
+    <namespace name="Revit.Schedules">
+      <category>Revit.Schedules</category>
+    </namespace>
```
Comment (QilongTang) [inline src/Libraries/RevitNodes/RevitNodes_DynamoCustomization.xml]: @Racel @mjkkirschner Could this be Revit.Views.ScheduleView so it appear with other schedule Node at the same sub category? Should these three nodes be prefixed with ScheduleView for consistency even though this creates long names?

**Comment by jnealb** [comment]
@mjkkirschner @QilongTang For example the ExportOptions that are needed for Export are under a different Revit subcategory, which is ackward.

**Pull request merged**